AUTHORS.rst

```diff
@@ -124,7 +124,7 @@ Patches and Suggestions
 - Bryce Boe <bbzbryce@gmail.com> (`@bboe <https://github.com/bboe>`_)
 - Colin Dunklau <colin.dunklau@gmail.com> (`@cdunklau <https://github.com/cdunklau>`_)
 - Bob Carroll <bob.carroll@alum.rit.edu> (`@rcarz <https://github.com/rcarz>`_)
-- Hugo Osvaldo Barrera <hugo@barrera.io> (`@hobarrera <https://github.com/hobarrera>`_)
```

Comment:
I'd rather not remove these. I don't think there's value in removing these links to folks even if they've deleted their account.

Reply:
Fair enough - I've removed all cases like this from the relevant commit.

I've left in the case where the GitHub user has changed, but let me know if you want me to remove that too.